common.blocks/i-bem/i-bem.bemhtml

```diff
@@ -425,9 +425,9 @@ def()(function() {
             }
 
             if(isBEM && jsParams) {
-                this._str += ' data-bem="' +
-                    this.attrEscape(JSON.stringify(jsParams)) +
-                    '"';
+                this._str += ' data-bem=\'' +
+                    this.xmlEscape(JSON.stringify(jsParams)).replace(/'/g, '&#39;') +
```

Comment:
+1 for sqAttrEscape for single quoted values as analog of attrEscape for double quoted.